ztools/scada/opcua/craft_packets.go

```diff
@@ -0,0 +1,100 @@
+package opcua
+
+import (
+	"encoding/hex"
+)
+// hello packet in hex
+var hexhel string = "48454c" +				// Message Type = Hel
+		    "46" +				// Chunk Type = F = Final
+		    "3e000000" +			// Message Size = 62
+		    "00000000" +			// Proto Version = 0
+		    "00000100" +			// ReceivedBufferSize = 65636
+		    "00000100" +			// SendBufferSize = 65636
+		    "00000001" +			// MaxMsgSize = 167
+		    "88130000" +			// MaxChunkCount = 5000
+	            "00000000"				// Endpoint URL = OPC Null String
+
+// function to build hello ; calculates message size if endpoint url is changed in the future
+func BuildHel() ([]byte,error){
+	hel , err := hex.DecodeString(hexhel[:8] + message_size(len(hexhel)/2) +  hexhel[16:])
```

Comment:
Can we split these constants into two (`hexhel[:8]` and `hexhel[16:]`), since they're not actually appended to each other?